### DISCUSSION TITLE
Unresolved reference: filament after updating to 1.44.0

### DISCUSSION THREAD

**Discussion by kyadalu1** [google/filament#7246] [Q&A] [answered]
After upgrading to 1.44.0 I am getting import errors for import com.google.android.filament,  what's the new import or are there any other breaking changes? I tried looking at the sample for android and its not updated yet

**Comment by romainguy** [comment]
What's unresolved? We haven't changed package names.

**Reply by kyadalu1** [reply]
<img width="497" alt="Screenshot 2023-10-09 at 10 28 25 AM" src="https://github.com/google/filament/assets/142881488/a83d351e-6fc8-406f-a5b3-0b5a0e725490">

I just added below two dependencies in my project

```
 implementation("com.google.android.filament:filament-utils-android:1.44.0")
  implementation("com.google.android.filament:filament-android:1.44.0")
```

If i switch to 1.43.1 then it works

**Reply by romainguy** [reply, marked as answer]
1.44 hasn't been published to Maven. You should be getting a Gradle error on sync.

@bejado

**Reply by kyadalu1** [reply]
cool thanks. Gradle sync worked fine for me for 1.44
